packages/mwp-app-render/src/components/seoHead.test.jsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import { SEOHeadComponent } from './SEOHead';
+
+const MOCK_PROPS = {
+	baseUrl: 'https://baseUrl.com',
+	imageUrl: 'https://www.mock-image-url.com/',
+	ldJson: [{ foo: 'MOCK_LD_JSON' }, { bar: 'MOCK_LD_JSON' }],
+	localeCode: 'es-ES',
+	pageDescription: 'mock description',
+	pageKeywords: 'mock keywords',
+	pageMeta: [{ mock: 'meta' }, { more: 'mock' }],
+	pageTitle: 'mock-page-title',
+	robots: false,
+	route: '/mock/preview/route',
+};
+
+const renderComponent = (props = MOCK_PROPS) => shallow(<SEOHeadComponent {...props} />);
+
+describe('SEOHead', () => {
+	it('renders component markup', () => {
+		expect(renderComponent()).toMatchSnapshot();
+	});
+	describe('robots', () => {
+		const robotsSelector = 'meta[name="robots"]';
+		it('defaults to robots "on"', () => {
+			const props = {...MOCK_PROPS};
+			delete props.robots;
+			expect(renderComponent(props).find(robotsSelector)).toMatchSnapshot();
+		});
+		it('explicity turns robots "on"', () => {
+			const props = {...MOCK_PROPS, robots: true};
+			delete props.robots;
+			expect(renderComponent(props).find(robotsSelector)).toMatchSnapshot();
+		});
+		it('turns robots "off"', () => {
+			const props = {...MOCK_PROPS, robots: false};
+			expect(renderComponent(props).find(robotsSelector)).toMatchSnapshot();
+		});
+	});
+	describe('Custom Open Graph data', () => {
+		const ogDescriptionSelector = 'meta[property="og:description"]';
+		it('renders pageDescription as open graph description', () => {
+			expect(renderComponent().find(ogDescriptionSelector)).toMatchSnapshot();
```

Comment:
this test is already part of the full default snapshot

Reply:
That's true, but I feel like there's a useful distinction, in that this explicitly calls this out with a snap that only looks at `og:description`. And where the 'full default snapshot' feels like it just snaps if things simply render, this delves specifically into its content a little more.

Reply:
it's useful to think about what the test would show if something changed unexpectedly - i.e. someone made an update that removed the `og:description` the snapshot test failure would highlight _just that markup_ (the diff would be specific), and the developer would have to figure out why it changed, knowing that it was an expected part of the original snapshot. That is nearly the same behavior as this test, only you get a more specific test description.

You should definitely think of snapshot as more than just "did this render", but "did the rendered object look exactly like this" so that you don't have to write tests for nested markup. By analogy, you currently have a check for `if (a && b)` and then you're checking for `if (a)` inside that block in order to be more explicit, which you generally wouldn't do.